### PR TITLE
feat: Generalized Google URL patterns to support more services

### LINKED ISSUE #18: Add Google Jamboard account switching
https://jamboard.google.com/u/0/

### PR DESCRIPTION
**Pull request by jaeyongjaykim**
- Closes #18 : Add Jamboard account switching
- ~Add Google Workspace, Admin, Search Console, Analytics support as well!~
  - Generalized Google URL patterns to support more services!
- ~Fixed India-specific domain (.co.in) to global (.com) domain.~

### CODE REVIEW

File: background.js [hunk 1]
```diff
@@ -88,24 +88,24 @@ chrome.runtime.onMessage.addListener(function (command) {
...
     console.log(current_url, shopping_regex);
     if (current_url.match(shopping_regex)) {
       update_url_regex = RegExp("authuser=[0-9]");
       update_acc = "authuser=" + account_num;
     }
 
-    // Google translate
-    const translate_regex = "https:\/\/translate.google.co.in\/.*\?authuser=[0-9].*";
+    // Google Translate
+    const translate_regex = "https:\/\/translate.google.com\/.*\?authuser=[0-9].*";
```
Comment (anshulahuja98) [inline background.js:108]: Can you please add compatibility of regions?
You can search for regex which accepts either .com
or other regional domain endings

Should most likely be a well known regex

**Review by anshulahuja98 — CHANGES_REQUESTED** [review]

File: background.js (hunk 1 above)
Comment (jaeyongjaykim) [reply background.js:108]:
```js
	const google_regex_type1 = ".*.google.*\/u\/[0-9].*"
    if (current_url.match(google_regex_type1)) {
      update_url_regex = RegExp("u\/[0-9]");
      update_acc = "u/" + account_num;
    }

    const google_regex_type2 = ".*.google.*\?authuser=[0-9].*"
    if (current_url.match(google_regex_type2)) {
      update_url_regex = RegExp("authuser=[0-9]");
      update_acc = "authuser=" + account_num;
    }
```

How about simplifying all codes to these two patterns? @anshulahuja98

Comment (anshulahuja98) [reply background.js:108]: I think that makes sense
Please go ahead and make the changes for this.

**Comment by jaeyongjaykim** [comment]
Review please 😊 @anshulahuja98

**Review by anshulahuja98 — APPROVED** [review]

**Comment by anshulahuja98** [comment]
LGTM.
Thank you for your contribution @jaeyongjaykim !

I'll shortly take a new release in a few days.

**Pull request merged**